event_model/__init__.py

```diff
@@ -1384,6 +1389,11 @@ def pack_datum_page(*datum):
     -------
     datum_page : dict
     """
+    if not datum:
+        raise ValueError(
+            "The pack_datum_page() function was called with empty *args. "
+            "Cannot create an DatumPage from an empty collection of Datum "
```

Comment:
```suggestion
            "Cannot create an DatumPage from an empty collection of Datums "
```

Reply:
"Datums" makes me cringe since the English (Latin, really) plural of "datum" is "data".

Reply:
"Datums" makes me cringe since the English (Latin, really) plural of "datum" is "data".

Reply:
I agree, thought about it, but wanted to make it consistent with the `EventPages` above. Let's leave it as is here.